examples/InfluxDB/Examples.InfluxDB/Program.cs

```diff
@@ -0,0 +1,68 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Exporter.InfluxDB;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+var appBuilder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+appBuilder.Services.AddControllers();
+
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+appBuilder.Services.AddEndpointsApiExplorer();
+appBuilder.Services.AddSwaggerGen();
+
+Action<ResourceBuilder> configureResource = r => r.AddService(
+    serviceName: "influx-exporter-test",
+    serviceVersion: typeof(Program).Assembly.GetName().Version?.ToString() ?? "unknown",
+    serviceInstanceId: Environment.MachineName);
+
+appBuilder.Services.AddOpenTelemetry()
```

Comment:
Could you add a simple Console application example instead of a full-blown AspNetCore application just to keep things easy?

Reply:
Done.